### PR TITLE
RF3/Py3 edits for Corner Tools, Ground Control and Interpolation Matrix

### PR DESCRIPTION
**Pull request by justvanrossum**
Let me know if anything needs changing.

### CODE REVIEW

File: CornerTools/CornerTools.roboFontExt/info.plist [hunk 1]
```diff
@@ -26,13 +26,13 @@
 	<key>name</key>
 	<string>CornerTools</string>
 	<key>requiresVersionMajor</key>
-	<string>1</string>
+	<string>3</string>
 	<key>requiresVersionMinor</key>
-	<string>4</string>
```
Comment (typemytype) [inline CornerTools/CornerTools.roboFontExt/info.plist]: this should be version 3

File: Interpolation Matrix/InterpolationMatrix.roboFontExt/info.plist [hunk 2]
```diff
@@ -26,13 +26,13 @@
 	<key>name</key>
 	<string>Interpolation Matrix</string>
 	<key>requiresVersionMajor</key>
-	<string>1</string>
+	<string>3</string>
 	<key>requiresVersionMinor</key>
-	<string>4</string>
```
Comment (typemytype) [inline Interpolation Matrix/InterpolationMatrix.roboFontExt/info.plist]: this should be version 3

File: CornerTools/CornerTools.roboFontExt/info.plist (hunk 1 above)
Comment (typemytype) [reply CornerTools/CornerTools.roboFontExt/info.plist]: sorry my mistake

**Comment by gferreira** [comment]
I think we can close this pull request without merging – RF3 versions can now be found in separate individual repositories @roboDocs. (links to the RF3 versions were added to the README)

can we [archive](http://help.github.com/articles/archiving-repositories/) this repository?

**Comment by justvanrossum** [comment]
Agreed on both points. I'll close this PR.

**Pull request closed without merging**